bridge_deployer/common/common.go

```diff
@@ -90,7 +47,14 @@ func GetLondonAccountAuth(client *ethclient.Client, addressPrivateKey string) *b
 
 	auth.Nonce = big.NewInt(int64(nonce))
 	auth.Value = big.NewInt(0) // in wei
-	//auth.GasLimit = uint64(12000000) // in units
+	if signing == Legacy {
+		gasPrice, err := client.SuggestGasPrice(context.Background())
+		if err != nil {
+			log.Fatal(err)
+		}
+		auth.GasLimit = uint64(12000000) // in units
```

Comment:
It looks like `auth.GasLimit` and `auth.GasPrice` are only set if `signing == Legacy` but not when `signing == London`. 

Is that expected?